docs/application/tizen-studio/common-tools/command-line-interface.md

```diff
@@ -674,7 +674,7 @@ cli-config, list, create, build-native, build-web, clean, certificate, security-
 
   Examples:
       Creates a template project based on the basic Tizen mobile UI project:
-          Windows®:
+          Windows&reg;:
```

Comment:
This line is inside of code style \```. So this line just shows the string as it as. (Windows\&reg;:)

Reply:
GitHub doesn't converts symbols  in code block. I deleted &reg;.